pyro/distributions/score_parts.py

```diff
@@ -13,7 +15,11 @@ def __mul__(self, scale):
         Scale appropriate terms of a gradient estimator by a data multiplicity factor.
         Note that the `score_function` term should not be scaled.
         """
-        log_pdf, score_function, entropy_term = self
-        return ScoreParts(log_pdf * scale, score_function, entropy_term * scale)
+        if is_identically_one(scale):
+            return self
+        log_pdf = scale_tensor(self.log_pdf, scale)
+        score_function = self.score_function
```

Comment:
i think this might be the wrong semantics for score_function for a scale that is purely a zero/ones mask.... i'm not sure
if we can stick mask semantics into poutine.scale without additional args/logic

Reply:
Good point. Is it sufficient to scale the `downstream_cost` factor, or does that still overcount in the non-batched case? https://github.com/uber/pyro/blob/ca8b05c/pyro/infer/tracegraph_elbo.py#L161